lib/search.rb

```diff
@@ -458,27 +458,13 @@ def self.advanced_filters
     end
   end
 
-  advanced_filter(/tags?:([a-zA-Z0-9,\-_+]+)/) do |posts, match|
-    if match.include?('+')
-      tags = match.split('+')
-
-      posts.where("topics.id IN (
-      SELECT tt.topic_id
-      FROM topic_tags tt, tags
-      WHERE tt.tag_id = tags.id
-      GROUP BY tt.topic_id
-      HAVING to_tsvector(#{default_ts_config}, array_to_string(array_agg(tags.name), ' ')) @@ to_tsquery(#{default_ts_config}, ?)
-      )", tags.join('&'))
-    else
-      tags = match.split(",")
+  advanced_filter(/^tags?:(.*)/) do |posts, match|
```

Comment:
Is there a reason that `([a-zA-Z0-9,\-_+]+)/)` has been changed to `(.*)`?

Reply:
I'll change it back.